### PR TITLE
Minor flagpatch typo fixes

### PR DESCRIPTION
**Pull request by Llywelwyn**
title. two spelling fixes + a small desc change where structure didn't quite make sense

### CODE REVIEW

**Comment by Llywelwyn** [comment]
!bugfix

**Comment by Llywelwyn** [comment]
!review

**Comment by listerla** [comment]
While you're at it; the /improper in the Gus flagpatch needs to be changed to \improper.

File: html/changelogs/llywelwyn-typos.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - spellcheck: "Fixed some grammar in the Jargon and Eridani flagpatches."
```
Comment (Llywelwyn) [inline html/changelogs/llywelwyn-typos.yml:41]:
```suggestion
  - spellcheck: "Fixed some grammar in the flagpatch loadout items."
```

**Review by KingOfThePing — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**